src/CmsAdmin/CategoryWidgetRelationController.php

```diff
@@ -140,16 +140,29 @@ public function toggleAction(Request $request)
      */
     public function sortAction(Request $request)
     {
+        $category = $this->getCategoryOrRedirect($request->categoryId);
         $this->getResponse()->setTypePlain();
         //brak pola
         if (null === $serial = $request->getPost()->__get('widget-item')) {
             return $this->view->_('controller.categoryWidgetRelation.move.error');
         }
         //sortowanie
-        (new \Cms\Model\CategoryWidgetModel($this->categoryId, $this->cmsSkinsetConfig))
+        (new \Cms\Model\CategoryWidgetModel($category->id, $this->cmsSkinsetConfig))
             ->sortBySerial($serial);
         //pusty zwrot
         return '';
     }
 
+    private function getCategoryOrRedirect(int $categoryId): CmsCategoryRecord
+    {
+        //wyszukiwanie kategorii
+        if ((null === $category = (new \Cms\Orm\CmsCategoryQuery)
```

Comment:
FQN

Reply:
no dobrze, ale to nie było zmieniane, przecież teraz nie zmienię w całym systemie :D